Sources/Runner/Script+Imports.swift

```diff
@@ -51,6 +59,24 @@ extension Script {
         let fileName = importPath.split(separator: "/").last
         return folder.path + "Sources/\(name)/\(fileName ?? "")"
     }
+    
+    private func generateXCodeProjWithConfig(configPath: String) throws {
```

Comment:
Would be better if we could pass to Marathon the xcconfig for the xcodeproj in the arguments, maybe worth propose that on the Marathon repo